docs/1.0.0/install_macos.md

```diff
@@ -0,0 +1,230 @@
+sudo easy_install pip
+```
+
+### Install virtualenv
+
+
+
+```
+pip3 install virtualenv
+```
+
+### Install libpq
+
+```
+brew install libpq
+```
+
+### Install postgresql
+
+```
+brew install postgresql
+```
+
+### Install psycopg2
+
+```
+env LDFLAGS="-I/usr/local/opt/openssl/include -L/usr/local/opt/openssl/lib" pip install psycopg2
+```
+
+
+## Build RIF Lumino from code
+
+1. Get the code from [https://github.com/rsksmart/lumino/releases/tag/1.0.0](https://github.com/rsksmart/lumino/releases/tag/1.0.0)
+2. Uncompress the downloaded file
+2. Go to the path you uncompressed the code in the previous step (lets call this path `$RIF_LUMINO_PATH`)
+3. Create python virtual env for RIF Lumino (this needs to be performed only once) and execute the following command:
+
+```
+virtualenv -p <PATH_TO_PYTHON3.7> clientEnv
+```
+
+**Note:**
+Replace `<PATH_TO_PYTHON3.7>` with the path where Python3.7 is installed in your system. In the case of MacOS, this is usually `/usr/local/bin/python3.7`
+
+4. Activate python virtual env, by executing the following command:
+
+```
+source clientEnv/bin/activate
+```
+
+5. Check if the Python version is correct inside the virtual environment by running:
+
+```
+python --version
+```
+
+This command should output version 3.7.x
+
+6. Install RIF Lumino requirements. Inside the virtual environment run the following command (this could take a few minutes):
+
+```
+pip install -c constraints.txt --upgrade -r requirements-dev.txt
+```
+
+7. Run Lumino setup with the following command:
+
+```
+python setup.py develop
+```
+
+## Start RIF Communications transport layer
+
+The comunication between Lumino nodes can be done both using RIF Communications (https://www.rifos.org/communications) and Matrix (https://matrix.org/).
+
+The default way, and the one we encourage to use for a more decentralized ecosystem, is RIF Communications. 
+
+In order to run Lumino using RIF Comms, you need to set up the RIF Communications bootnode. To use Matrix, no configuration is required. 
+
+### Set up a RIF Communications bootnode
+
+#### Requirements:
+
+- Node 14+ (14.13 recommended)
+
+#### Get the code
+
+- git clone https://github.com/rsksmart/rif-communications-pubsub-bootnode/tree/grpc-api 
+- run npm install 
+
+
+#### Create a key for the RIF Communications bootnode
+
+- Create the `config/keys/lumino` folder
+- execute `openssl ecparam -genkey -name secp256k1 -out ec_key.pem -param_enc explicit` and then `openssl pkcs8 -in ec_key.pem -topk8 -v2 aes-256-cbc -v2prf hmacWithSHA256 -outform DER -out ec_key_pkcs8_v2.der` and define a password
+
+#### Create a config file
+
+- go back to `config` folder
+- edit the `lumino.json5` file. modify the `key.password` value to your key's password
+- change the `key.privateKeyURLPath` to the config key path `[...]/config/keys/lumino/ec_key_pkcs8_v2.der`
+
+#### Start the node
+
+- At root folder, run `NODE_ENV=lumino npm run api-server`
+- You should see that the GRPC Api started on port `5013` 
```

Comment:
We should add a log output here

Reply:
here you go:

You should see a log output similar to this one:

```shell
[INFO] 11:34:08 ts-node-dev ver. 1.1.1 (using ts-node ver. 9.1.1, typescript ver. 4.1.3)
Loading encrypted DER key
Node started, listening on addresses:
/ip4/127.0.0.1/tcp/5011/p2p/16Uiu2HAmQswXYkmzDTgs2Em5JkhP8Y33CEhXQUHY3trctB1StTe7
/ip4/127.0.0.1/tcp/5012/ws/p2p/16Uiu2HAmQswXYkmzDTgs2Em5JkhP8Y33CEhXQUHY3trctB1StTe7

Listening on topics: 


PEERID: 16Uiu2HAmQswXYkmzDTgs2Em5JkhP8Y33CEhXQUHY3trctB1StTe7
GRPC Server started on port 5013
```

Reply:
thanks!